### PR TITLE
Configure SFML to work on windows and linux

### PR DESCRIPTION
**Pull request by srijan-paul**
This PR mainly changes two things:
 - `CMakeLists.txt`: link SFML to the target and configure includes.
 -  `main.cpp` : A simple dummy SFML window to test if things are working correctly.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -14,7 +16,9 @@ file(GLOB_RECURSE SOURCES CONFIGURE_DEPENDS "src/*.cpp")
 # to that library.
 add_executable(${PROJECT_NAME} ${SOURCES})
 
-target_include_directories(${PROJECT_NAME} PRIVATE "include")
+target_include_directories(${PROJECT_NAME} PRIVATE include)
+target_include_directories(${PROJECT_NAME} PRIVATE "SFML/include")
```
Comment (karnkaul) [inline CMakeLists.txt]: Is this needed/even correct? (Shouldn't it be `ext/SFML/...`?)

File: src/main.cpp [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+			switch (event.type) {
+			case sf::Event::Closed: window.close(); break;
+			case sf::Event::KeyPressed: std::cout << "ping\n"; break;
+			default: break;
+			}
+		}
+		window.clear();
+		window.display();
+	}
+	return EXIT_SUCCESS;
```
Comment (karnkaul) [inline src/main.cpp:22]: Just FYI (no need to change): in C++ we usually just write `0` (or omit `return 0` entirely).

**Review by karnkaul — APPROVED** [review]
Recommend considering `.editor-config` to take care of line endings, terminating newline, file encoding, etc.

File: src/main.cpp (hunk 2 above)
Comment (srijan-paul) [reply src/main.cpp:22]: Yeah, I just wanted to be a bit explicit. This will most likely go away anyway.

**Review by karnkaul — APPROVED** [review]

**Pull request merged**